src/pages/VideoListing/VideoListing.jsx

```diff
@@ -0,0 +1,48 @@
+import React from "react";
+import styles from "./videolisting.module.css";
+import { v4 as uuid } from "uuid";
+import { Navbar, Sidebar, VideoCard, Chips } from "components";
+
+export function VideoListing() {
+  const videos = [
+    {
+      _id: uuid(),
+    },
+    {
+      _id: uuid(),
+    },
+    {
+      _id: uuid(),
+    },
+    {
+      _id: uuid(),
+    },
+  ];
+  return (
+    <div className={styles.container}>
+      <section className={styles.navbar}>
+        <Navbar />
+      </section>
+      <section className={styles.sidebarNav}>
+        <Sidebar />
+      </section>
+      <section className={styles.main}>
+        <div className={styles.categoryChips}>
+          {[
+            "Matches",
+            "Badminton Tutorials",
+            "Badminton Trick Shots",
+            "Players Interview",
+          ].map((categoryName) => (
```

Comment:
can make a variable for category names or in backend we have categories in DB you can fetch categories from there as well

Reply:
Yes, I will do that once I integrate the backend. Using it till the time I fetch data from APIs.